config.go

```diff
@@ -42,13 +42,13 @@ func (v Volume) String() string {
 	return ""
 }
 
-type Envs []Env
-type Env struct {
+type Pairs []Pair
+type Pair struct {
 	Name  *string `yaml:"name,omitempty"`
 	Value *string `yaml:"value"`
 }
 
-func (e Env) String() string {
+func (e Pair) String() string {
```

Comment:
The name of the type has been change to something more generic than `Env` but stringer still contains `Env \"%s\"` in fmt string

Reply:
Good catch!